### PR TITLE
Microsoft SQL Server/v1

### PR DESCRIPTION
**Pull request by erinkcochran87**
This PR:

- Adds documentation for the v1 (Singer) Microsoft SQL Server integration
- Refactors the way data types are displayed and stored

### CODE REVIEW

**Comment by erinkcochran87** [comment]
See the JIRA ticket for outstanding questions.

- **Data types**: [Preview](https://deploy-preview-336--stitchdocs.netlify.com/docs/integrations/databases/microsoft-sql-server#data-types) | Files - [MSSQL-specific](https://github.com/stitchdata/docs/pull/336/files#diff-ccf251108de7b402d1c5f6543bdb5eda), [v1](https://github.com/stitchdata/docs/pull/336/files#diff-cf1b66df4ab90e7dfcbfcc703a4d6c0f)

- **Setup guide**
  - **Requirements**: [Preview](https://deploy-preview-336--stitchdocs.netlify.com/docs/integrations/databases/microsoft-sql-server/v1#setup-requirements) | [File](https://github.com/stitchdata/docs/pull/336/files#diff-3cc6a4b14894cc6f130c08f9a1e34fb7R73)
  - **Enabling Change Tracking**: [Preview](https://deploy-preview-336--stitchdocs.netlify.com/docs/integrations/databases/microsoft-sql-server/v1#enable-log-based-incremental-change-tracking) | [File](https://github.com/stitchdata/docs/pull/336/files#diff-7425a8e22d65f8a5c6d1e25a468bed9c)

File: _data/connect/common/database-sources.yml [hunk 1]
```diff
@@ -125,7 +138,7 @@ binlog-replication:
     description: |
       If `true`, Log-based Incremental Replication will be used as the default Replication Method for tables set to replicate. This choice can be overriden on individual tables at any time.
```
Comment (dmosorast) [inline _data/connect/common/database-sources.yml]: Want to make note of this, as far as connect is concerned, does the tap need to do anything to respect this? or does the API fill it in on selection. I know the UI will fill it in, but not sure about Connect.

File: _database-integrations/mssql/amazon-rds/v1/microsoft-sql-server-amazon-rds-v1.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+        - Create/manage Security Groups, which is required to whitelist Stitch's IP addresses.
+        - View database details, which is required for retrieving the database's connection details.
+  - item: "**Privileges in {{ integration.display_name }} that allow you to create/manage users.** This is required to create the Stitch database user."
+  - item: |
+      **If using Log-based Incremental Replication**, you'll need:
+
+      - **A database running Microsoft SQL Server {{ page.log-based-replication-minimum-version }} or higher.** Earlier versions of Microsoft SQL Server don't include Change Tracking functionality which is required for Log-based Incremental Replication.
+      - **The `ALTER DATABASE` privilege in Microsoft SQL Server.** This is required to complete the setup for Log-based Incremental Replication.
+  - item: |
+      **A database that uses case-insensitive collation**. Refer to [Microsoft's documentation]({{ site.data.taps.links.mssql.collation }}){:target="new"} for more info.
```
[the remark this answers is not in the document]
Comment (dmosorast) [reply _database-integrations/mssql/amazon-rds/v1/microsoft-sql-server-amazon-rds-v1.md]: Would defer to Harrison for confirmation of this.

File: _data/taps/extraction/data-types/mssql/v1.yml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  doc-link: *int-doc-link     
+
+time:
+  stitch-type: "string"
+  format: "date-time"
+  doc-link: &time-doc-link "https://docs.microsoft.com/en-us/sql/t-sql/data-types/time-transact-sql?view=sql-server-2017"
+
+timestamp:
+  stitch-type: "string"
+  format: "date-time"
```
[the remark this answers is not in the document]
Comment (erinkcochran87) [reply _data/taps/extraction/data-types/mssql/v1.yml]: Good to know!

File: _database-integrations/mssql/azure/v1/microsoft-azure-v1.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+      {% include integrations/templates/create-database-user-tabs.html %}
+
+  - title: "Connect Stitch"
+    anchor: "connect-stitch"
+    content: |
+      In this step, you'll complete the setup by entering the database's connection details and defining replication settings in Stitch.
+
+    substeps:
+      - title: "Define the database connection details"
+        anchor: "define-connection-details"
```
[the remark this answers is not in the document]
Comment (erinkcochran87) [reply _database-integrations/mssql/azure/v1/microsoft-azure-v1.md]: I updated the copy to reflect this (I think). I was able to connect an Azure DB successfully, though. I'm gonna table this, and we can come back to it if need be.

**Pull request merged**